.changeset/rare-geese-trade.md

```diff
@@ -0,0 +1,12 @@
+---
+"@swapkit/toolbox-substrate": major
+"@swapkit/wallet-talisman": major
+"@swapkit/helpers": major
+"@swapkit/wallets": major
+"@swapkit/core": major
+"@swapkit/plugin-evm": major
+"@swapkit/api": major
+"@swapkit/sdk": major
+---
+
+add talisman wallet package
```

Comment:
it shouldn't be major - at max it's minor on sdk and wallets, rest is minor

Reply:
Yeah I updated, helpers does not need patch, also most other packages as well